controller/helper/process-survey-instances.js

```diff
@@ -102,14 +102,22 @@ function calculateTimeLeft (openTime, endTime, completedTime) {
 
     // calculate the time in hours until end time
     const totalAvailibleTime = endTime.diff(openTime, 'hours');
-    const timeTaken = endTime.diff(completedTime, 'hours');
+    let percentTimeLeft = minTime;
+    let timeTaken = minTime;
 
-    // caculate percent of time taken out of total time availible to take the survey
-    const percentTimeLeft = Math.round(timeTaken / totalAvailibleTime * percent);
+    if (completedTime !== null && !isNaN(completedTime)) {
+        timeTaken = endTime.diff(completedTime, 'hours');
```

Comment:
remove `let timeTaken = minTime;` from line 106, and make this line (109) `const`.